project4/index.html

```diff
@@ -0,0 +1,197 @@
+
+            <!-- speakers -->
+            <h3 id="speakers">Keynote Speakers</h3>
+            <section class="right_border">
+                <div class="speaker_wrapper">
+                    <div class="speaker_card">
+                        <img class="speaker-image" src="images\asset3.png"
+                            alt="a photo of scott thomas speaking at a podium">
+                        <h4>Scott Thomas</h4>
+                        <p>As the Design Director for Obama’s 2008 campaign, Scott Thomas led a now-historic political
+                            campaign, in which branding, design, and the web played a truly pivotal role. Likening the
+                            experience to “building an airplane in flight,” Scott talks about the creative’s need for
+                            triage, the crucial role of incremental design improvements, and the importance of getting
+                            back to the hand and keeping things simple.
+                        </p>
+                        <h4 class="talk_title">Talk: Making Obama</h4>
+                        <p class="talk_details">How did an obscure senator rise to the highest office in the land and
+                            celebrity status? With the aid of branding and design. Scott details his involvement in
+                            Obama's campaign on Day 1.
+                        </p>
+                        <!-- https://99u.adobe.com/videos/5821/scott-thomas-designing-the-obama-campaign -->
+                    </div>
+                    <div class="speaker_card">
+                        <img class="speaker-image" src="images\asset2.png"
+                            alt="a photo of sam adaramola sitting during an interview">
+                        <h4>Sam Adaramola</h4>
+                        <p>Born in Nigeria, raised in America, Sam is currently a full-time digital designer and
+                            marketer.
+                            He
+                            has a
+                            degree in Advertising and Public relations, and also a studied film as minor in school.
+                            Formerly
+                            an
+                            account executive at a global PR/Advertising firm, Sam currently works as a freelance
+                            creative.
+                        </p>
+                        <h4 class="talk_title">Talk: Why Young People Feel the "Bern"</h4>
+                        <p class="talk_details">Why do so many young people love someone way over the hill? Student loan
+                            debt? Charisma? Weed? Sam Adaramola knows and he will share it on Day 2.
+                        </p>
+                        <!-- http://afropolitanyouth.org/2016-aya-career-exploration-day/-->
+                    </div>
+                    <div class="speaker_card">
+                        <img class="speaker-image" src="images\asset1.png"
```

Comment:
```suggestion
                        <img class="speaker-image" src="images/asset1.png"
```
![it was effort to find this many Slash gifs that were not inappropriate](https://media.giphy.com/media/l0HUg59QUApfsuIWk/giphy.gif)